torch/onnx/symbolic_opset16.py

```diff
@@ -51,6 +51,11 @@ def grid_sampler(
     padding_mode_enum,
     align_corners,
 ):
+    # Check the input and grid tensor rank beforehand.
+    if len(input.shape) == 5:
```

Comment:
`input` is a symbolic value of type `torch.Value`, so you'd need
https://github.com/pytorch/pytorch/blob/d41b5d7c145f3e09c7223c2b707933266241ec9b/torch/onnx/symbolic_opset9.py#L1021-L1022
to get its rank.

Reply:
Thank you. BTW, could you please also educate me where I can find the definition of `torch._C.Value`?

Reply:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/OVERVIEW.md#value is a good place to start.